pyth-rng/src/command/run.rs

```diff
@@ -0,0 +1,95 @@
+use {
+    crate::{
+        api,
+        config::RunOptions,
+        ethereum::PythContract,
+        state::{
+            HashChainState,
+            PebbleHashChain,
+        },
+    },
+    anyhow::{
+        anyhow,
+        Result,
+    },
+    axum::{
+        routing::get,
+        Router,
+    },
+    std::{
+        error::Error,
+        sync::Arc,
+    },
+    tower_http::cors::CorsLayer,
+    utoipa::OpenApi,
+    utoipa_swagger_ui::SwaggerUi,
+};
+
+pub async fn run(opts: &RunOptions) -> Result<(), Box<dyn Error>> {
+    #[derive(OpenApi)]
```

Comment:
I'm not sure why rustfmt doesn't catch these but they're formatted so roughly here, I can catch these later but It's a shame they don't auto-format, I guess It's to do with being embedded within the derive.